src/README.md

```diff
@@ -2,30 +2,29 @@
 
 ### Table of Contents
 
--   [consts][1]
```

Comment:
Funny, if I recall correctly, quite recently Kuba M. changed `documentation` version and the formatting also changed. 
Looks like there's a lot stuff ongoing in that package ;-) 

(All good, no changes expected, just sharing a thought)

Reply:
Well, yeah :smile: They don't like `-` now :smile: